lib/LedgerSMB/Admin/Command/user.pm

```diff
@@ -112,20 +115,20 @@ sub _option_spec {
             # employee
```

Comment:
Where is `permission` applied? Or if it's applied, where is `role` applied?!

Reply:
Cleaned confusion between user role (Job title) and postgres roles (permissions)